src/main/java/ru/svgogin/service/spark/service/SparkService.java

```diff
@@ -50,12 +80,25 @@ private Company toEntity(CompanyDto companyDto) {
   }
 
   private CompanyDto toDto(Company company) {
-    return new CompanyDto(company.getInn(),
+    return new CompanyDto(
+        company.getInn(),
         company.getOgrn(),
         company.getKpp(),
         company.getFullNameRus(),
         company.getShortNameRus(),
         company.getStatusName(),
         company.getStatusDate());
   }
+
+  private Company actualize(Company company, CompanyDto companyDto) {
+    return new Company(
+        company.getId(),
+        companyDto.getInn() != null ? companyDto.getInn() : company.getInn(),
```

Comment:
you can use `Objects.requireNonNullElse`:
```suggestion
        requireNonNullElse(companyDto.getInn(), company.getInn()),
```